index.html

```diff
@@ -1275,6 +1549,101 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
 encoding flexibility. The benefits of the LD-Proof approach are flexibility at
 the cost of implementation complexity.
           </dd>
+          <dt id="pf20">
+PF20: Based on existing official standards.
+          </dt>
+          <dd>
+JSON and JWT are proposed and mature IETF standards. While JSON-LD 1.0 is in
+REC state in W3C, JSON-LD 1.1 is still in WD state. LD-Proofs are not standardized at all.
+          </dd>
+          <dt id="pf21">
+PF21: Designed to be small in size.
+          </dt>
+          <dd>
+JSON was invented as a simple data format to be transmitted on the wire. A verifiable credential
+can be expressed by its attributes only, without the necessity to introduce additional meta-information
+such as @context. This makes the resulting JSON+JWT credential typically also smaller in size.
+          </dd>
+          <dt id="pf22">
+PF22: Offline support without further processing.
+          </dt>
+          <dd>
+A JWT can fully describe itself without the need to retrieve or verify any external documents.
+JSON-LD requires the context to be queryable and requires further documents to be accessible
+to check the prevalent document, e.g., LD-Proof. Additional caching needs to be implemented
+to support offline use cases.
+          </dd>
+          <dt id="pf23">
+PF23: Wide adoption in other existing standards.
+          </dt>
+          <dd>
+JWT founds its application in many other existing standards, e.g., OAuth2, OpenID Connect.
+This allows for backward compatibility with existing authentication and authorization
+frameworks without or with only minor modifications to these legacy systems.
+          </dd>
+          <dt id="pf24">
+PF24: No type ambiguity.
+          </dt>
+          <dd>
+It is best practice that JSON data structures typically do not expect changing types of
+their internal attributes. JSON-LD has implicit support for compact form serialization
+which transforms arrays with a single element only to switch its data type. Developers
+writing parsers have to implement special handling of these data types, which results in
+more code, is more error-prone and sometimes does not allow parsers based on code
+generation, which rely on static types.
```

Comment:
Any JSON data must first pass a validation step before making these sorts of assumptions and any constraints can be enforced with either format (e.g., via JSON schema) ... so I don't really understand this point as a differentiator. Furthermore, this is about the differences in the proof format, not what's in the VC data model itself. Even then, a specific profile of a VC would have to be enforced via a schema.

Reply:
@dlongley 

> Made some minor suggestions.
> Why are the checklist and responses duplicated in the implementation guide?

@brentzundel I don't know. That had happened in the previous version. I was also wondering why this is required. We could think about addressing this issue in a separate PR where we refactor that for JWTs and LD-Proofs. For the time being, we should leave it as is.